Makefile

```diff
@@ -41,6 +45,7 @@ endif
 
 ifeq ($(BUILD_NUMBER),dev)
 	export MM_FEATUREFLAGS_GRAPHQL = true
+	GOTAGS += "testlicensekey"
```

Comment:
Is this the case for spinwicks and cloud test servers ? They may have a build number set

Reply:
build number is only `dev` for local development. A separate change in [build-ee](https://git.internal.mattermost.com/mattermost/ci/build-ee/-/tree/MM-50182) will control the mm-ee-test image

Reply:
Gotcha! That makes sense 👍